src/utils/file.utils.ts

```diff
@@ -103,6 +103,15 @@ export const checkFileTypeExists = (
 	return { exists: existsSync(filePath), filePath };
 };
 
+export const getEndOfLine = (fileContent: string) => {
```

Comment:
The code of this function can be shortened. You can return the result in one line.
```js
return fileContent.split(/\r\n|\r|\n/).length
```